src/app.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { AppService, KeyValuePair } from './app.service';
+
+@Controller()
+export class AppController {
+  constructor(private readonly appService: AppService) {}
```

Comment:
@satikaj , have you spend some time learning the concept call dependency injection ?